Assets/_Project/Scripts/UserInterface/Components/List/ListPage.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace PolSl.UrbanHealthPath.UserInterface.Components.List
+{
+    public class ListPage : MonoBehaviour
+    {
+        [SerializeField] private List<ListPanelElement> panelElements = new List<ListPanelElement>();
+
+        private const int MaxElementsCount = 3;
+
+        private void Awake()
+        {
+            foreach (ListPanelElement panelElem in panelElements)
+            {
+                if (panelElem.gameObject.activeSelf)
+                    panelElem.gameObject.SetActive(false);
+            }
+        }
+
+        public void Initialize(List<ListElement> elements)
+        {
+            for (int i = 0; i < MaxElementsCount; i++)
+            {
+                if (i < elements.Count)
+                {
+                    ListElement element = elements[i];
+                    ListPanelElement panelElem = panelElements[i];
+
```

Comment:
unnecessary empty line.

Reply:
IMO increases readability, so it's fine.